tests/js/jest.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+	rootDir: '../../',
+	moduleDirectories: [ 'node_modules', '<rootDir>/client' ],
+	moduleNameMapper: {
+		'^react$': '<rootDir>/node_modules/react',
+		'^react-dom$': '<rootDir>/node_modules/react-dom',
+		'^moment$': '<rootDir>/node_modules/moment',
```

Comment:
Ugh, I lost like 2 hours debugging this, because I restarted this PR from scratch and didn't copy these 2 lines. How much time did it take you to figure out that the issue was `moment` being resolved as 2 different packages?

Reply:
I wish I new, but I totally understand you. I remember some of those issues took pretty much time to find and fix.